bot/role_management/role_management.py

```diff
@@ -131,6 +130,203 @@ async def module_role_error(self, ctx: commands.Context, error: commands.Command
+
+        Allows the bot owner to manage so called "reaction roles" for messages in the configured role channel. This can
+        be done via multiple subcommands like `add`, `remove` or `clear`.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+        """
+        await ctx.send_help(ctx.command)
+
+    @reaction_role.command(name="add")
+    @commands.is_owner()
+    @command_log
+    async def add_reaction_role(self, ctx: commands.Context, message: discord.Message, emoji: str, role: discord.Role):
+        """Command Handler for the subcommand `add` of the `reactionrole` command.
+
+        Adds a reaction to a specified message and creates a corresponding database entry for it to work as a so called
+        reaction role.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            message (discord.Message): The message to which the reaction role should be added.
+            emoji (str): The emoji of the reaction which will be added.
+            role (discord.Role): The specific role a member should get when adding the reaction.
+        """
+        if message.channel.id != self.ch_role.id:
+            await ctx.send(f":information_source: Reaction-Roles können nur für Nachrichten im Kanal "
+                           f"{self.ch_role.mention} erstellt werden.")
+            return
+        if emoji in [reaction.emoji for reaction in message.reactions]:
+            await ctx.send(":x: Für den angegebenen Emoji existiert bereits eine Reaction-Role.")
+            return
+
+        self._db_connector.add_reaction_role(message.id, emoji, role.id)
+        await message.add_reaction(emoji)
+        log.info("A reaction role has been added to the message with id %s.", message.id)
+
+        await ctx.send(":white_check_mark: Die Reaction-Role wurde erfolgreich erstellt.")
+
+    @reaction_role.command(name="remove")
+    @commands.is_owner()
+    @command_log
+    async def remove_reaction_role(self, ctx: commands.Context, message: discord.Message, emoji: str):
+        """Command Handler for the subcommand `remove` of the `reactionrole` command.
+
+        Removes the reaction from a specified message and deletes the corresponding database entry of the reaction role.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            message (discord.Message): The message from which the reaction role should be removed.
+            emoji (str): The emoji of the reaction.
+        """
+        if message.channel.id != self.ch_role.id:
+            await ctx.send(f":information_source: Nachrichten außerhalb des Kanals {self.ch_role.mention} können keine "
+                           f"Reaction-Roles besitzen.")
+            return
+        if emoji not in [reaction.emoji for reaction in message.reactions]:
```

Comment:
```
from operator import attrgetter #is part of the STL

...

if emoji not in map(attrgetter('emoji'), message.reactions)
```

This is supposedly faster according to [SO](https://stackoverflow.com/questions/45477181/apply-map-over-property-in-python) bc it executes all natively in C

Reply:
Maybe in comparison to the code provided by the user who asked the question. In our case, it's much slower:
![image](https://user-images.githubusercontent.com/49726903/93670372-38521600-fa9b-11ea-9dd4-67b862b8f051.png)